src/x/instrument/types.go

```diff
@@ -78,4 +80,10 @@ type Options interface {
 
 	// CustomBuildTags returns the custom build tags.
 	CustomBuildTags() map[string]string
+
+	// SetProfilerOptions sets the profiler options.
+	SetProfilerOptions(value profiler.Options) Options
+
+	// ProfilerOptions returns the ProfilerOptions.
+	ProfilerOptions() profiler.Options
```

Comment:
I think the `profiler.Options` could be inlined here, i.e. these could be `SetProfiler()` and `Profiler()` instead.

Reply:
Agree. Now it is only used for setting profiler so we could inline it.